plugins/getting-started.md

```diff
@@ -93,9 +92,9 @@ Plugins communicate with Penpot by sending and receiving messages through the if
 @startuml
 
 skinparam state {
-    BackgroundColor    transparent
-    BorderColor        black
-    ArrowColor         black
+BackgroundColor transparent
```

Comment:
Did you test that this change is not breaking the graph?

Reply:
I checked and the whitespaces don't affect the graph, thanks!